### PR TITLE
Branch creation process2

### PR DESCRIPTION
**Pull request by karthikjava23**

### CODE REVIEW

**Comment by aravindchinna** [comment]
@karthikjava23 can you please review and merge

**Review by karthikjava23 — COMMENTED** [review]
Changes are okey

File: PreviousComapnyName.txt [hunk 1]
```diff
@@ -1 +1,5 @@
+My Previous company is Capgemini.
+hello
+
```
Comment (aravindchinna) [inline PreviousComapnyName.txt]: remove unnecessary spaces

**Review by aravindchinna — CHANGES_REQUESTED** [review]

File: PreviousComapnyName.txt (hunk 1 above)
Comment (karthikjava23) [reply PreviousComapnyName.txt]: Removed spaces, can you please review it again

**Review by aravindchinna — APPROVED** [review]

**Pull request merged**